xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/macros.vm

```diff
@@ -287,6 +287,9 @@
   <div id="${htmlLiveTableId}-inaccessible-docs" class="hidden">
     #info($services.localization.render('rightsmanager.documentrequireviewrights'))
   </div>
+  <div id="${htmlLiveTableId}-computed-title-docs" class="hidden">
+    #info("(&#xb9;)&#160;$services.localization.render('platform.livetable.doc_title_computed.filter_sort_hint')")
```

Comment:
Since you're generating HTML I think it would be more clear to use ``<sup>1</sup>`` and ``&nbsp;`` (for non-breaking space) rather than these cryptic HTML code entities.

And regarding the translation key, I haven't seen that many keys using underscore. We're using camel case (or dot) most of the time I think.

Reply:
I see about the translation key; will fix it this evening.
I am not sure about the marker; I can use the ``<sup>1</sup>`` as well, but then I should use it in the display of the entries as well. (The other marker telling about non-viewable pages currently uses a plain ``*``)

Reply:
@mflorea there are many translation keys with underscore but maybe this is not the best practice. In fact [translations keys from old core](https://github.com/xwiki/xwiki-platform/blob/master/xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties) are not very consistent (mix of camel case, underscore, dots and lower case).

Reply:
Yeah many of the oldcore translation are much older that any kind of best practice :)

See https://dev.xwiki.org/xwiki/bin/view/Community/DevelopmentPractices#HTranslationPropertyNaming.

Reply:
I went the rout to use CSS to create that symbol, so it will remain a "cryptic code entity" but this time in the CSS (to avoid encoding issues).